multinetwork.php

```diff
@@ -221,4 +173,91 @@ function mangle_url( $url, $path, $orig_scheme, $site_id ) {
 	$mangled = preg_replace( $regex, '\1' . $current_mapping->get_domain(), $url );
 
 	return $mangled;
+}
+
+/**
+ * Get all possible mappings which may be in use and apply to the supplied domain
+ *
+ * This will return an array of domains which might have been mapped but also apply to the current domain
+ * i.e. a given url of site.network.com should return both site.network.com and network.com
+ *
+ * @param $domain
+ * @param null $honor_www whether or not we should include www and non variants of the domains
+ * @return array
+ */
+function get_possible_mapped_domains( $domain, $honor_www = null ) {
+
+	if ( $honor_www === null ) {
+		/**
+		 * Filter whether or not we should honor www
+		 *
+		 * By default we ignore use of www. vs no www
+		 *
+		 * @param bool - whether or not to honor www
```

Comment:
See above comment about param docs; also, first param takes the var name being assigned (so `$honor_www` here)